routes/truckRoutes.js

```diff
@@ -1 +1,105 @@
-// put your Express router code in here
\ No newline at end of file
+'use strict';
+// put your Express router code in here
+
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-undef */
+
+var express = require('express');
+var foodTrucks = require('../trucks');
+
+var router = express.Router();
+
+var trucks = require('../trucks');
+var foodTypes = require('../trucks');
+var openTrucks = require('../trucks');
+var typeTrucks = require('../trucks');
+/* eslint-enable no-undef */
+
+// var serveStatic = express.static('public');
+
+var axeTruck = 'Fez';
+
+var openTruck = trucks.filterTrucksByDay('Saturday');
+var pizzaTruckList = trucks.filterTrucksByFoodType('pizza');
+var addTruck = trucks.addTruck();
+var deleteTruck = trucks.removeTruck(axeTruck);
+
+/*
+.use(function (request, response) {
+  console.log('Request URL: ', request.originalURL);
+})
+.use(function (request, response) {
+  console.log('Time: ', Date.now());
+})
+*/
+
+/* eslint-disable no-console */
+
+router.route('/')
+.get(function (request, response) {
```

Comment:
better to indent when chaining methods like this - otherwise it's difficult to tell what's going on